tap_netsuitesuiteql/tap.py

```diff
@@ -54,7 +54,7 @@ def discover_streams(self) -> list[streams.NetsuiteSuiteQLStream]:
         """
         return [
             streams.ArrHistoryStream(self),
-            streams.TransactionLineStream(self),
+            # streams.TransactionLineStream(self),
```

Comment:
Any reason for disabling this ?

Reply:
We actually don't use this data

Reply:
It significantly speeds up the meltano update.